frontend/src/classes/TailwindElement.ts

```diff
@@ -0,0 +1,28 @@
+import { LitElement } from "lit";
+
+// Load the contents of `theme.css`, processed with postcss (tailwind), as a
+// string using Webpack's raw-loader.
+// TODO(emma, 2023-12-12) Figure out how this'll need to change with Webpack 5,
+// as it removes raw-loader
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const themeCSS = require("!!raw-loader!postcss-loader!../theme.css").default;
```

Comment:
_Theoretically_ we could use this same stylesheet on the document root too, and avoid even having this in there twice

Reply:
It should work without raw-loader + same stylesheet in document now, try this out: https://github.com/webrecorder/browsertrix-cloud/pull/1452/files

Reply:
Oh sick, thank you!